make-nagios-fhservices-cfg.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+
+fh_services_ping = ['fh-messaging', 'fh-metrics']
+fh_services_health = ['fh-messaging', 'fh-metrics']
+
+rhmap_admin_email = os.getenv('RHMAP_ADMIN_EMAIL', 'root@localhost')
+rhmap_router_dns = os.getenv('RHMAP_ROUTER_DNS', 'rhmap.localhost')
+
+template_file = '/opt/rhmap/fhservices.cfg.j2'
+nagios_config_filename = '/etc/nagios/conf.d/fhservices.cfg'
+
+template_basename = os.path.basename(template_file)
+template_dirname = os.path.dirname(template_file)
+
+j2env = Environment(loader=FileSystemLoader(template_dirname), trim_blocks=True)
+j2template = j2env.get_template(template_basename)
+
+j2renderedouput = j2template.render(fh_services_ping = fh_services_ping,
+                                    fh_services_health = fh_services_health,
```

Comment:
Minor thing, for consistency with the two lines below (and Python's PEP8), please remove the spaces around the equal sign in the lines above.